searx/autocomplete.py

```diff
@@ -162,7 +163,7 @@ def stract(query, _lang):
     if not resp.ok:
         return []
 
-    return [suggestion['raw'] for suggestion in resp.json()]
+    return [html.unescape(suggestion['raw']) for suggestion in resp.json()]
```

Comment:
We already have a method for that:
```suggestion
    return [html_to_text(suggestion['raw']) for suggestion in resp.json()]
```
see https://github.com/searxng/searxng/blob/39aaac40d63d53555b7fc12d56f0825c85ecf567/searx/utils.py#L149